ConsoleGUI/Controls/VerticalScrollPanel.cs

```diff
@@ -105,19 +113,16 @@ void IDrawingContextListener.OnUpdate(DrawingContext drawingContext, Rect rect)
 
 		void IInputListener.OnInput(InputEvent inputEvent)
 		{
-			switch (inputEvent.Key.Key)
+			if (inputEvent.Key.Key == ScrollUpKey)
 			{
-				case ConsoleKey.UpArrow:
-					Top -= 1;
-					return;
```

Comment:
It looks like a bug in the old code - that it does `return` instead of a `break` for the `ConsoleKey.UpArrow`.